charts/darts-portal/values.yaml

```diff
@@ -3,11 +3,10 @@ nodejs:
   image: 'sdshmctspublic.azurecr.io/darts/portal:latest'
   ingressHost: darts-portal.{{ .Values.global.environment }}.platform.hmcts.net
   aadIdentityName: darts
-  #  Uncomment once the vault containing the app insights key has been set up
-  #  keyVaults:
-  #    darts:
-  #      secrets:
-  #        - name: AppInsightsInstrumentationKey
-  #          alias: azure.application-insights.instrumentation-key
+  keyVaults:
+    darts:
+      secrets:
+        - name: AppInsightsInstrumentationKey
+          alias: APP_INSIGHTS_KEY
```

Comment:
Hi @davet1985, I notice that you're not using the AppInsightsConnectionString secret here. Is it not required? Thanks

Reply:
@alancoleman I've just looking this up as I didn't know. They are interchangeable, and in fact the connection string is preferred as the keys are being phased out by end of March 2025.